EIPS/eip-1.md

```diff
@@ -37,11 +37,31 @@ An EIP must meet certain minimum criteria. It must be a clear and complete descr
 
 ## EIP Work Flow
 
+### Shepherding an EIP
+
 Parties involved in the process are you, the champion or *EIP author*, the [*EIP editors*](#eip-editors), and the [*Ethereum Core Developers*](https://github.com/ethereum/pm).
 
-:warning: Before you begin, vet your idea, this will save you time. Ask the Ethereum community first if an idea is original to avoid wasting time on something that will be rejected based on prior research (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Ethereum is used. Examples of appropriate public forums to gauge interest around your EIP include [the Ethereum subreddit], [the Issues section of this repository], and [one of the Ethereum Gitter chat rooms]. In particular, [the Issues section of this repository] is an excellent place to discuss your proposal with the community and start creating more formalized language around your EIP.
+Before you begin writing a formal EIP, you should vet your idea. Ask the Ethereum community first if an idea is original to avoid wasting time on something that will be be rejected based on prior research. Proposing an EIP which has not been properly dilligenced may be grounds for EIP Editors not granting the "Draft" status. It is thus recommended to open a discussion thread on [the Ethereum Magicians forum] to do this, but you can also use [one of the Ethereum Gitter chat rooms], [the Ethereum subreddit] or [the Issues section of this repository]. 
+
+In addition to making sure your idea is original, it will be your role as the champion to socialize your EIP and build community consensus around it. You should try and gauge whether the interest in your EIP is commensurate with both the work involved in implementing it and how many parties will have to conform to it. For example, the community interest required for a technically complex Core EIP will have to be much greater than for a simple ERC. While "community interest" will not be a factor in deciding whether an EIP gets merged as a draft, insufficient community interest may prevent your EIP from moving forward, especially if it is a Core EIP. 
```

Comment:
@MadeofTin another edit based on your feedback, re: "community interest" not block for Draft, but potentially blocking afterwards. 

> While "community interest" will not be a factor in deciding whether an EIP gets merged as a draft, insufficient community interest may prevent your EIP from moving forward, especially if it is a Core EIP.